assets/sass/components/global/_googlesitekit-help-menu.scss

```diff
@@ -41,4 +39,33 @@
 			width: 100%;
 		}
 	}
+
+	.googlesitekit-help-menu {
+		box-shadow: 0 4px 12px 0 rgba(0, 0, 0, 0.15), 0 2px 4px 0 rgba(0, 0, 0, 0.15);
+
+		.mdc-list {
+			padding: 12px 0;
+		}
+
+		.googlesitekit-cta-link svg {
+			fill: none;
+		}
+
+		.googlesitekit-cta-link {
+			letter-spacing: 0.25px;
+			line-height: 20px;
+		}
+
+		.googlesitekit-cta-link .googlesitekit-icon-wrapper {
+			margin-right: 16px !important;
```

Comment:
Please add a comment to explain why `!important` is needed here, as per my suggestion below.